paper/paper.md

```diff
@@ -70,19 +70,19 @@ Analysis of electrophysiological recordings is commonly started by multiple prep
 
 # Statement of need
 
-We aim to simply the pre-processing and feature-estimation using those invasive modalities. To simply this analysis, *py_neuromodulation* utilizes two parametrization files: *settings.yaml* and *channels.tsv*, that include all required steps for reproducibility. It allows for a streamlined pipeline of pre-processing, feature estimation and decoding. The performance contribution of various pre-processing steps can be assessed in combination with different feature estimation modalities and machine learning model architectures.  
-Offline analysis often includes whole-recording normalization, non-causal filtering, or feature computation with test-set data leakage [@merk_machine_2022], which limits real-time compatibility. In py_neuromodulation, the data-source is represented as a Python generator, and can be interchangeable for online and offline processing, meaning that there is no difference in handling data streamed directly from a brain implant, or stored from an archival dataset. This enables training of generalizable models that are directly applicable to online applications, using exactly the same algorithms. 
-Finally, electrophysiological research entails multiple technical and conceptual hurdles that pose substantial entry barriers for scientists without software development training. By providing an intuitive GUI for parametrization and raw-data and feature visualization, we aim to lower the domain knowledge requirements for entering this field.
-Py_neuromodulation was already used in several scientific projects with various applications: ECoG and DBS-LFP movement decoding [@merk_electrocorticography_2022;@cavallo_reinforcement_2025], movement intention classification [@kohler_dopamine_2024], seizure classification [@merk_invasive_2023], emotional valence decoding [@merk_invasive_2023], gait symptom estimation in rodents [@elgarulli_elgarullineurokin_2025], amongst others.
+We aim to simplify the pre-processing and feature-estimation using those invasive modalities. To this end, *py_neuromodulation* utilizes two parametrization files: *settings.yaml* and *channels.tsv*, that include all required steps for reproducibility. It allows for a streamlined pipeline of pre-processing, feature estimation and decoding. The performance contribution of various pre-processing steps can be assessed in combination with different feature estimation modalities and machine learning model architectures.  
```

Comment:
I don't understand the last sentence: "The performance contribution of various pre-processing steps can be assessed in combination with different feature estimation modalities and machine learning model architectures." - do you provide a functionality that somehow checks the performance?